src/drivers/fluid_adriver.c

```diff
@@ -460,18 +460,16 @@ delete_fluid_audio_driver(fluid_audio_driver_t* driver)
  * for the usage with fluidsynth.
  * The \c names are the same as being used for the \c audio.driver setting.
  * 
- * By default all audio drivers fluidsynth has been compiled with are registered.
+ * By default all audio drivers fluidsynth has been compiled with are registered, so calling this function is optional.
  * 
- * Any attempt of using audio drivers that have not been registered is undefined behaviour!
+ * @warning This function may only be called if no thread is residing in fluidsynth's API and no instances of any kind
+ * are alive (e.g. as it would be the case right after fluidsynth's inital creation). Else the behaviour is undefined.
+ * Furtermore any attempt of using audio drivers that have not been registered is undefined behaviour!
  * 
```

Comment:
I'm regularly doing the sames typos errors ("behavior") line 466 and 467 :)

Reply:
`Behaviour` is British English, `Behavior` American. British is the correct English ;P

Reply:
Good to know.